samples/boards/intel_s1000_crb/audio/audio_app.rst

```diff
@@ -7,15 +7,17 @@ Overview
 ********
 
 This sample application demonstrates audio capture and playback on an
-Intel® S1000 CRB.
+Intel® Sue Creek S1000 CRB with control from a Linux host connected
```

Comment:
Don't use "Intel®" with a code name (e.g., Sue Creek).  The legal name is "Intel® Speech Enabling Developer Kit".  I'd recommend not mentioning Sue Creek.  For that matter S1000 isn't approved with an Intel® before it either.

I'd recommend just writing this name as ``Intel S1000 CRB``.

Reply:
Intel(R) Sue Creek S1000 is the now legal approved naming. Intel(R) S1000 is not correct, hence the change.
This is not the developer kit.

Reply:
Hmmm... OK.  That change isn't noted in the Intel trademark db (yet).